changelog/Microsoft.Partner.CustomerSvcAdmin.json

```
@@ -0,0 +1,150 @@
+          "ChangedApiName": "delegatedAdminAccessAssignmentStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminAccessAssignmentStatus** enumeration type.",
+          "Target": "delegatedAdminAccessAssignmentStatus"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Enum Type",
+          "ChangedApiName": "delegatedAdminAccessContainerType",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminAccessContainerType** enumeration type.",
+          "Target": "delegatedAdminAccessContainerType"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Enum Type",
+          "ChangedApiName": "delegatedAdminRelationshipOperationStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipOperationStatus** enumeration type.",
+          "Target": "delegatedAdminRelationshipOperationStatus"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Enum Type",
+          "ChangedApiName": "delegatedAdminRelationshipOperationType",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipOperationType** enumeration type.",
+          "Target": "delegatedAdminRelationshipOperationType"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Enum Type",
+          "ChangedApiName": "delegatedAdminRelationshipRequestAction",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipRequestAction** enumeration type.",
+          "Target": "delegatedAdminRelationshipRequestAction"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Enum Type",
+          "ChangedApiName": "delegatedAdminRelationshipRequestStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipRequestStatus** enumeration type.",
+          "Target": "delegatedAdminRelationshipRequestStatus"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Enum Type",
+          "ChangedApiName": "delegatedAdminRelationshipStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipStatus** enumeration type.",
+          "Target": "delegatedAdminRelationshipStatus"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "delegatedAdminAccessContainer",
+          "ChangeType": "Addition",
+          "Description": "Added the [delegatedAdminAccessContainer](https://docs.microsoft.com/en-us/graph/api/resources/delegatedAdminAccessContainer?view=graph-rest-beta) resource type.",
```

Suggestion to enhance the description
```suggestion
          "Description": "Added the [delegatedAdminAccessContainer](https://docs.microsoft.com/en-us/graph/api/resources/delegatedAdminAccessContainer?view=graph-rest-beta) resource type. Use this resource to define the details of the security group in a Microsoft partner's tenant through which delegated access to the customer tenant will be assigned.",
```